src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BOUND = 100;
+    private final Ball ball = new Ball();
+    private final ColorSupplier newColor = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        int randomNumber = random.nextInt(BOUND);
+        ball.setNumber(randomNumber);
+        ball.setColor(newColor.getRandomColor());
+        return ball;
+    }
+}
```

Comment:
The use of static methods is discouraged according to the checklist. Instead of using a static final constant for BOUND, consider using an instance variable if this value is supposed to change, or keep it static final if it is a constant for all instances of the class.